### DISCUSSION TITLE
Modal for Attaching Resources in Relationship Fields

### DISCUSSION THREAD

**Discussion by muhammadsaeedparacha** [laravel/nova-issues#6066] [Ideas & Feature Requests]
When I am Attaching a Relation through any Relationship Field
Such as Belongs To, Tag etc.
I should be able to open a modal
The modal shows the relationships in an index view
Where I can search, sort, filter results
And I can see the full table instead of only title/subtitle,
And then I can click to attach the relationship

Example from odoo:
<img width="503" alt="Screenshot 2023-11-22 at 7 17 14 PM" src="https://github.com/laravel/nova-issues/assets/2680384/82fb6519-026e-4471-a367-d3d5ff81931f">
<img width="1013" alt="Screenshot 2023-11-22 at 7 19 16 PM" src="https://github.com/laravel/nova-issues/assets/2680384/f43f5879-6c3c-4494-9177-66867a127ad9">

**Comment by msucevan** [comment]
Yes would be interesting, any update on that?
